q2_diversity/_ordination.py

```diff
@@ -9,5 +9,10 @@
 import skbio.stats.ordination
 
 
-def pcoa(distance_matrix: skbio.DistanceMatrix) -> skbio.OrdinationResults:
-    return skbio.stats.ordination.pcoa(distance_matrix)
+def pcoa(distance_matrix: skbio.DistanceMatrix, method: str = 'eigh',
```

Comment:
Being that this is a high-level interface (unlike scikit-bio), I would suggest that the function signature be simplified to:

```python
def pcoa(distance_matrix: skbio.DistanceMatrix, number_of_dimensions: int=None) -> skbio.OrdinationResults
    if number_of_dimensions is None:
        # calculate full decomposition using eigh
        # using inplace=False
    else:
        #  calculate the decomposition only for the `number_of_dimensions` using fsvd
        # using inplace=True

    return ordination      
```

The logic being that it doesn't make a lot of sense to select a subset of dimensions for a full decomposition and that when that is done, we might as well just avoid the extra work.

Sorry if I'm coming in too late to the discussion.

Reply:
I like this even better!